src/containers/notification/components/TwitterShare.tsx

```diff
@@ -151,41 +179,52 @@ export const getRemixCosignText = async (
   }
 }
 
-export const getRewardsText = (notification: ChallengeReward) => ({
+const getRewardsText = async (notification: ChallengeReward) => ({
   text: `I earned $AUDIO for completing challenges on @AudiusProject #AudioRewards`,
   link: null
 })
 
-export const getNotificationTwitterText = async (notification: any) => {
-  if (notification.type === NotificationType.Milestone) {
-    return getAchievementText(notification)
-  } else if (notification.type === NotificationType.TrendingTrack) {
-    return getTrendingTrackText(notification)
-  } else if (notification.type === NotificationType.RemixCreate) {
-    return getRemixCreateText(notification)
-  } else if (notification.type === NotificationType.RemixCosign) {
-    return getRemixCosignText(notification)
-  } else if (notification.type === NotificationType.ChallengeReward) {
-    return getRewardsText(notification)
-  }
+const tierInfoMap: Record<BadgeTier, { label: string; icon: string }> = {
+  none: { label: 'None', icon: '' },
+  bronze: { label: 'Bronze', icon: '🥉' },
+  silver: { label: 'Silver', icon: '🥈' },
+  gold: { label: 'Gold', icon: '🥇' },
+  platinum: { label: 'Platinum', icon: '🥇' }
```

Comment:
is the platinum icon here correct? also looks like gold

Reply:
yea I don't think there's another emoji for medals so the design just had gold twice haha